domain-server/resources/describe-settings.json

```diff
@@ -347,8 +347,8 @@
-               "span": 11
+              "label": "Permissions <a data-toggle='tooltip' data-html='true' title='<p><strong>Domain-Wide User Permissions</strong></p><ul><li><strong>Connect</strong><br />Sets whether a user can connect to the domain.</li><li><strong>Rez Avatar Entities</strong><br />Sets whether a user can display avatar entities.</li><li><strong>Lock / Unlock</strong><br />Sets whether a user change the &ldquo;locked&rdquo; property of an entity (either from on to off or off to on).</li><li><strong>Rez</strong><br />Sets whether a user can create new entities.</li><li><strong>Rez Temporary</strong><br />Sets whether a user can create new entities with a finite lifetime.</li><li><strong>Rez Certified</strong><br />Sets whether a user can create new certified entities.</li><li><strong>Rez Temporary Certified</strong><br />Sets whether a user can create new certified entities with a finite lifetime.</li><li><strong>Write Assets</strong><br />Sets whether a user can make changes to the domain&rsquo;s asset-server assets.</li><li><strong>Ignore Max Capacity</strong><br />Sets whether a user can connect even if the domain has reached or exceeded its maximum allowed agents.</li><li><strong>Replace Content</strong><br>Sets whether a user can replace entire content sets by wiping existing domain content.</li><li><strong>Get and Set Private User Data</strong><br>Sets whether a user can get and set the privateUserData entity property.</li></ul><p>Note that permissions assigned to a specific user will supersede any parameter-level permissions that might otherwise apply to that user. Additionally, if more than one parameter is applicable to a given user, the permissions given to that user will be the sum of all applicable parameters. For example, let&rsquo;s say only localhost users can connect and only logged in users can lock and unlock entities. If a user is both logged in and on localhost then they will be able to both connect and lock/unlock entities.</p>'>?</a>",
```

Comment:
instead of “can display avatar entities” everywhere, maybe “can create avatar entities”?

also, are we actually totally deleting the avatar entities?  or are we just not sending them to other clients?  it’s hard to say what is right: you should probably be seeing what everyone else sees, but it would also be nice if they came back if you receive permissions or move to a domain where you have permissions

Reply:
out of respect for peoples clothing, I also wonder if we need a prompt on the client side when you connect to a domain you don’t have avatar entity rez perms for.  eg “connecting to this domain will remove your avatar entities.  are you sure you want to continue?”

Reply:
We're not deleting avatar entities; we're just not displaying them. In the current code, they're not being distributed to other clients - but your avatar entities will currently still display for you (on e of the "not dones" is to not display them for yourself, either. ... So if you go to another domain where you do have permissions, everyone will see them again.

I think users would find it very annoying if we deleted their avatar entities and then they had to reattach them when they visited a different domain.

Wording: Instead of "can display avatar entities" perhaps "can rez avatar entities" would be better. (Users aren't actually "creating" avatar entities - they're just rezzing/displaying them, or not.)

I've added a third "not done" regarding warning users if they're rezzing in a domain that doesn't allow their avatar entities.

Reply:
hmm...does this break physics?  can avatar entities collide?  if only I’m simulating physics for my avatar entities, what if something bounces off of them?  what do other people see?

Reply:
Probably easiest to implement not rezzing your own avatar entities than sorting out physics. I'll have a look at this.

Reply:
Note that unless there is a protocol change, users will be able to use 2021.1 Interface release which won't hide a user's own avatar entities. So there's potential for any physics they're simulating to include their avatar entities.